inc/class_parser.php

```diff
@@ -1060,6 +1060,12 @@ function mycode_parse_url($url, $name="")
 		{
 			$nofollow = " rel=\"nofollow\"";
 		}
+		
+		// Fix eventual XSS attempts
+		if(strpos($url, 'javascript:') !== false)
```

Comment:
Shouldn't this check for position 0? Otherwise URLs like `http://example.com/javascript:something` might be changed too.